### PR TITLE
Improve Set Up Instructions & Environment Variable Descriptions

### PR DESCRIPTION
**Pull request by BorhanSaflo**
This PR improves the descriptions of environment variables and setup instructions to make it easier for developers to correctly set up the project.

### CODE REVIEW

**Review by jere-mie — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -25,42 +27,47 @@ This repository contains the source code for the University of Windsor Computer
...
+AZURE_AD_CLIENT_SECRET=""
+AZURE_AD_TENANT_ID=
+
+# Discord Configuration (Optional for Development - Required to link a user's account with their Discord, retrieve server information, and invite the user to the Discord server)
+DISCORD_CLIENT_ID=""
+DISCORD_CLIENT_SECRET=""
+DISCORD_BOT_TOKEN=""
+DISCORD_GUILD_ID=""
+DISCORD_CALLBACK_URL="http://localhost:3000/api/discord/callback"
+DISCORD_SUGGESTION_WEBHOOK_URL=""
```
Comment (tanzimfh) [inline README.md]: do we need to duplicate `.env.example` in the README?

Comment (BorhanSaflo) [reply README.md]: It's kind of redundant, should we just keep the `.env.example` file and remove it form the README?

Comment (tanzimfh) [reply README.md]: yes

**Review by tanzimfh — APPROVED** [review]

**Pull request merged**